### PR TITLE
[#IOPID-583] create ioweb portal cdn

### PR DESCRIPTION
**Pull request by arcogabbo**
### List of changes
* added CDN for ioweb
* added new resource group for the CDN

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-platform-iac-projects](https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build?view=folders)
2. select PR branch
3. wait for approval

### CODE REVIEW

File: .github/workflows/static_analysis.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           pids=()
 
-          TAG=$(cat .terraform-version)
+          TAG=1.5.6
```
Comment (michaeldisaro) [inline .github/workflows/static_analysis.yml:35]: I think we should update .terraform-version instead.

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

File: .github/workflows/static_analysis.yml (hunk 1 above)
Comment (arcogabbo) [reply .github/workflows/static_analysis.yml]: fix present in https://github.com/pagopa/io-infra/commit/44102f1819c531a50be0f27a72a7ebc60824391e

File: src/domains/ioweb-common/06_cdn.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+# keyvault where the TLS certificate is stored
+data "azurerm_resource_group" "profile_rg" {
+  name = format("%s-profile-sec-rg", local.product)
+}
+
+data "azurerm_key_vault" "profile_kv" {
+  name                = format("%s-profile-kv", local.product)
+  resource_group_name = data.azurerm_resource_group.profile_rg.name
+}
```
Comment (gquadrati) [inline src/domains/ioweb-common/06_cdn.tf:19]: Change it as soon as https://github.com/pagopa/io-azure-devops/pull/22 is merged

Comment (arcogabbo) [reply src/domains/ioweb-common/06_cdn.tf]: fixed in https://github.com/pagopa/io-infra/pull/661/commits/a0afe88cf40ca2215e21d36c92b64a3967135e08

**Review by michaeldisaro — APPROVED** [review]

**Review by gquadrati — APPROVED** [review]

**Pull request merged**